features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_bootstrap.c

```diff
@@ -923,6 +923,7 @@ void thread_interface_init(protocol_interface_info_entry_t *cur)
 
 }
 
+#ifdef MBED_CONF_TARGET_ENABLE_FLOATING_POINT
```

Comment:
We need to understand more what the impact of removing functions that use floating points is.